### ISSUE TITLE
Update EDA rulebook schema - rulebooks now located under `/extensions/eda/rulebooks`

### ISSUE THREAD

**Issue by alisonlhart** [ansible/ansible-lint#3283]
The EDA content structure has been changed by the EDA project team. Rulebooks will now be located under `/extensions/eda/rulebooks` instead of just `/rulebooks`. 

The `rulebook` schema and tests need to be updated to reflect this change. 

@thedoubl3j @benthomasson

**Comment by ssbarnea** [comment]
@alisonlhart Are you 100% sure that rulebooks are never expected to be found in a path that does not match `extensions/eda/rulebooks`?

I am asking this because current path also matches the proposed one and if we change the pattern, it will reduce the matches and linter will not identify files under a random `rulebooks/` folder as being rulebooks, it will treat them as any other YAML files.

**Comment by alisonlhart** [comment]
@ssbarnea We've been told this is a finalized change and that all rulebooks will be located under the `/extensions/eda/rulebooks` directory specifically, but if that's already covered by this it may be better to leave things as they are just in case. Since this is a completely new form of content, it could be ideal to account for more possible conditions rather than less.

**Comment by ssbarnea** [comment]
Lets not do anything about this until we have the false positive identification of a rulebook.

**Comment by minsis** [comment]
Can this be reopened?

It seems the official layout for rulebooks is now `extensions/eda/rulebooks`. Ansible lint now thinks files placed in this directory are playbooks and not rulebooks.

https://docs.ansible.com/projects/rulebook/en/latest/collections.html
https://github.com/ansible/event-driven-ansible/tree/main/extensions/eda


```
% ansible-lint extensions/eda/rulebooks/configure_repository_permissions.yml

yntax-check[specific]: 'match_multiple_rules' is not a valid attribute for a Play
extensions/eda/rulebooks/configure_repository_permissions.yml:1:3


# Rule Violation Summary

  1 syntax-check profile:min tags:core,unskippable

Failed: 1 failure(s), 0 warning(s) on 1 files. Profile 'production' was required.
```
